openstack/nova/templates/seed.yaml

```diff
@@ -71,9 +71,9 @@ spec:
         role: cloud_compute_admin
       - project: admin
         role: cloud_compute_admin
-    - name: {{ .Values.global.nova_service_user | default "nova" }}{{ .Values.global.user_suffix | default "" }}
+    - name: '{{ .Values.global.nova_service_user | default "nova" | include "resolve_secret" }}'
       description: Nova Service
-      password: {{ required ".Values.global.nova_service_password is missing" .Values.global.nova_service_password }}
+      password: '{{ required ".Values.global.nova_service_password is missing" .Values.global.nova_service_password | include "resolve_secret" }}'
```

Comment:
According to the secrets-injector documentation, it works only for "kind: Secret". So, I suppose, we have a problem with all OpenStack Seeds that create users.

Reply:
No. https://github.wdf.sap.corp/cc/secrets-injector#other-resource-kinds
>  In Converged Cloud controlplanes this is configured for kind OpenstackSeed.

I also validated the same with Erik.